config/routes.rb

```diff
@@ -399,12 +399,27 @@
   scope '/services' do
     mount AppsApi::Engine, at: '/apps'
     mount VBADocuments::Engine, at: '/vba_documents'
-    mount AppealsApi::Engine, at: '/appeals'
     mount ClaimsApi::Engine, at: '/claims'
     mount Veteran::Engine, at: '/veteran'
     mount VAForms::Engine, at: '/va_forms'
     mount VeteranVerification::Engine, at: '/veteran_verification'
     mount VeteranConfirmation::Engine, at: '/veteran_confirmation'
+
+    mount AppealsApi::Engine, at: '/appeals'
```

Comment:
If at all possible I'd like to keep the root routes clean w/ the `mount` for the module only, and keep the subpaths inside the module's routes.rb.

Reply:
Hmm, I had been assuming you couldn't `mount` multiple engines at the same point (which is why I avoided mounting at `/`), but it seems like that's possible - I'll try doing by mounting the appeals engine at `/` instead of `/appeals`

Reply:
D'oh, that was me forgetting the paths the segmented APIs were using. I think you can actually use the same engine for multiple routes, so it should be possible to:
```
mount AppealsApi::Engine, at: '/higher-level-reviews'
mount AppealsApi::Engine, at: '/notice-of-disagreements'
# ...etc...
```
and then our version paths, etc, can all be in the appeals_api module's routes file :D

Reply:
Thank you! This works great - I've rebased against master and applied this pattern.

I left the concerns in their own dedicated file for now, because I think it makes it easier to notice that/why these are mounted in multiple places for now - I do think we should get rid of that file and move everything back into `routes.rb` once we can get rid of the concerns, though.

Reply:
Actually, I spoke too soon - there's something wrong with the hyphenated routes here, I'll update again soon

Reply:
Ok - what do you think of this most recent commit?

When mounting the engine multiple times, I found that I wasn't sure how to differentiate between sub-routes used in multiple APIs (like the 'v0' or 'docs') without collision as long as the top-level `appeals`,  `higher-level-reviews`, `supplemental-claims`, etc. segments were 'out of our control' in the main routes.rb file, if that makes any sense.

I tried mounting the engine at `/` instead and managed to get the roots correct, but I had to use a `scope` directive to make sure the rails router didn't expect us to move all of our controller files into modules that match the path segments (that might actually be nice, but it would be too many lines changed for one PR)